docs/syntax_and_semantics/default_and_named_arguments.md

```diff
@@ -1,6 +1,6 @@
-## Default values
+## Default arguments
```

Comment:
I would leave this section title as is per https://github.com/crystal-lang/crystal/issues/10374#issuecomment-775140541 ff.